agent/bench-scripts/tests/pbench-fio/test-03.txt

```diff
@@ -15,5 +15,5 @@
 [debug][1900-01-01T00:00:00.000000] [pbench-fio] fio-3.21 is installed
 --- pbench.log file contents
 +++ test-execution.log file contents
-/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/require-rpm fio 3.21 gte
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/rpm --query --queryformat=%{EVR} fio
```

Comment:
This log now shows the mock for `rpm` instead of the mock for `require-rpm` because the test is now using the "real" `require-rpm` instead of the mock'd one and that invokes the mock'd `rpm`?